src/partition/src/error.rs

```diff
@@ -33,26 +34,26 @@ pub enum Error {
         source: meta_client::error::Error,
     },
 
-    #[snafu(display("Failed to find Datanode, table: {} region: {:?}", table, region))]
+    #[snafu(display("Failed to find Datanode, table id: {}, region: {}", table_id, region))]
     FindDatanode {
-        table: String,
+        table_id: TableId,
```

Comment:
When we troubleshoot based on log information, can we easily find the table_name according to the table_id?

Reply:
We can't, but it's the price we ought to pay when we enjoy the invariant and other benifits that table id gives. Luckily, we have admin APIs in metasrv that could get the table's info easily, or just get the raw values in etcd. On the other hand, in my experience, talking with table id or table name make little difference during debugging. They both "meaningless" to us who care little about business. Image that a log says "table orc_165 not found" against "table 42 not found".